android_p/google_diff/cel_kbl/system/bt/0004-Disable-Wideband-Support.patch

```diff
@@ -0,0 +1,28 @@
+From f03427de120934b830d8526e39ad03c8ba153b61 Mon Sep 17 00:00:00 2001
+From: Aiswarya Cyriac <aiswarya.cyriac@intel.com>
+Date: Thu, 7 Mar 2019 15:07:03 +0530
+Subject: [PATCH] Disable Wideband Support
+
+Change-Id: I7cfbed0736f9442daa4ec574701d9ba6cc0cd709
+Tracked-On: OAM-75510
+Signed-off-by: Aiswarya Cyriac <aiswarya.cyriac@intel.com>
+---
+ bta/hf_client/bta_hf_client_at.cc | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/bta/hf_client/bta_hf_client_at.cc b/bta/hf_client/bta_hf_client_at.cc
+index 5bcc68ecb..8e71bc037 100644
+--- a/bta/hf_client/bta_hf_client_at.cc
++++ b/bta/hf_client/bta_hf_client_at.cc
+@@ -1647,7 +1647,7 @@ void bta_hf_client_send_at_bac(tBTA_HF_CLIENT_CB* client_cb) {
+ 
+   APPL_TRACE_DEBUG("%s", __func__);
+ 
+-  buf = "AT+BAC=1,2\r";
++  buf = "AT+BAC=1\r";
+ 
+   bta_hf_client_send_at(client_cb, BTA_HF_CLIENT_AT_BAC, buf, strlen(buf));
+ }
+-- 
+2.17.1
+
```

Comment:
I see lots of checkpatch errors on this driver patch...


>> total: 67 errors, 70 warnings, 1066 lines checked

Please fix applicable errors.